elastic/kibana
# [Infra] Use asset details locators in the dashboards tab

Closes #180202 

## Summary

This PR adds asset details locators in the dashboards tab.

## Testing
- Create 2 dashboards: the second one should link to the first one (using links - the second one should open the Dashboard in a new tab) Example:
<img width="1918" alt="image" src="https://github.com/elastic/kibana/assets/14139027/e7b6485e-b923-4724-a8f8-26fc9a75b112">

- Go to the Dashboard tab inside the single host flyout 
- https://github.com/elastic/kibana/assets/14139027/bb668e7d-2751-4e22-a9b8-236b26313d72

- Go to the Dashboard tab inside the asset details full-page view: 
- https://github.com/elastic/kibana/assets/14139027/77f62e5d-45fc-4f4e-85b5-d4a1b03f76f4

- After navigating the correct dashboard should be shown and the time range should be the same
